test_kv_query_result.sh

```diff
@@ -11,26 +11,39 @@ HOME_2=${NEUTRON_DIR}/data/test-2/
 
 USERNAME_1=demowallet1
 USERNAME_2=demowallet2
-KEY_2=$(neutrond keys show demowallet2 -a --keyring-backend test --home ${HOME_2})
-ADMIN=$(neutrond keys show demowallet1 -a --keyring-backend test --home ${HOME_1})
+KEY_2=$(neutrond keys show ${USERNAME_2} -a --keyring-backend test --home ${HOME_2})
+ADMIN=$(neutrond keys show ${USERNAME_1} -a --keyring-backend test --home ${HOME_1})
 
 TARGET_ADDRESS=neutron1mjk79fjjgpplak5wq838w0yd982gzkyf8fxu8u
 VAL2=neutronvaloper1qnk2n4nlkpw9xfqntladh74w6ujtulwnqshepx
 
 # Upload the queries contract
+echo "Upload the queries contract"
 RES=$(${BIN} tx wasm store ${CONTRACT} --from ${USERNAME_1} --gas 50000000  --chain-id ${CHAIN_ID_1} --broadcast-mode=block --gas-prices 0.0025stake  -y --output json  --keyring-backend test --home ${HOME_1} --node tcp://127.0.0.1:16657)
 QUERIES_CONTRACT_CODE_ID=$(echo $RES | jq -r '.logs[0].events[1].attributes[0].value')
 echo $RES
 echo $QUERIES_CONTRACT_CODE_ID
 
 # Instantiate the queries contract
+echo "Instantiate the queries contract"
 INIT_QUERIES_CONTRACT='{}'
 
 RES=$(${BIN} tx wasm instantiate $QUERIES_CONTRACT_CODE_ID "$INIT_QUERIES_CONTRACT" --from ${USERNAME_1} --admin ${ADMIN} -y --chain-id ${CHAIN_ID_1} --output json --broadcast-mode=block --label "init"  --keyring-backend test --gas-prices 0.0025stake --home ${HOME_1} --node tcp://127.0.0.1:16657)
 echo $RES
 QUERIES_CONTRACT_ADDRESS=$(echo $RES | jq -r '.logs[0].events[0].attributes[0].value')
 echo $QUERIES_CONTRACT_ADDRESS
 
+# Send coins from ADMIN to QUERIES_CONTRACT_ADDRESS to perform register_interchain_query message
+echo "Send coins from ${ADMIN} to ${QUERIES_CONTRACT_ADDRESS} to perform register_interchain_query message"
+echo $(${BIN} tx bank send ${ADMIN} ${QUERIES_CONTRACT_ADDRESS} 10000000stake -y --chain-id ${CHAIN_ID_1} --output json --broadcast-mode=block --gas-prices 0.0025stake --gas 300000 --keyring-backend test --home ${HOME_1} --node tcp://127.0.0.1:16657)
+
+
 # Register a query for KEY_2 balance
+echo "Register a query for ${KEY_2} balance"
 RES=$(${BIN} tx wasm execute $QUERIES_CONTRACT_ADDRESS "{\"register_balance_query\": {\"connection_id\": \"connection-0\", \"denom\": \"stake\", \"addr\": \"${KEY_2}\", \"update_period\": 5}}" --from ${USERNAME_1}  -y --chain-id ${CHAIN_ID_1} --output json --broadcast-mode=block --gas-prices 0.0025stake --gas 1000000 --keyring-backend test --home ${HOME_1} --node tcp://127.0.0.1:16657)
-echo $RES
+echo $RES | jq
+
+# Register a query for KEY_2 balance
+echo "Register a query for total supply"
+RES=$(${BIN} tx wasm execute $QUERIES_CONTRACT_ADDRESS "{\"register_bank_total_query\": {\"connection_id\": \"connection-0\", \"denom\": \"stake\", \"update_period\": 5}}" --from ${USERNAME_1}  -y --chain-id ${CHAIN_ID_1} --output json --broadcast-mode=block --gas-prices 0.0025stake --gas 1000000 --keyring-backend test --home ${HOME_1} --node tcp://127.0.0.1:16657)
```

Comment:
```suggestion
RES=$(${BIN} tx wasm execute $QUERIES_CONTRACT_ADDRESS "{\"register_bank_total_supply_query\": {\"connection_id\": \"connection-0\", \"denom\": \"stake\", \"update_period\": 5}}" --from ${USERNAME_1}  -y --chain-id ${CHAIN_ID_1} --output json --broadcast-mode=block --gas-prices 0.0025stake --gas 1000000 --keyring-backend test --home ${HOME_1} --node tcp://127.0.0.1:16657)
```
That's not the only problem here by the way. The field "denom" is also wrong.